jupyterlab_comments/src/git.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ServerConnection } from '@jupyterlab/services';
+import { URLExt } from '@jupyterlab/coreutils';
+
+
+/*
+Send a request to the server
+Params:
+    handler = name of the handler route as defined in the jupyterlab_comments module
+    method = GET or POST
+    fileName = file path relative to the root of the Jupyter Lab server
+    currentPath = path from home directory to root of Jupyter Lab server
```

Comment:
This description seems inaccurate. This value doesn't seem like it is always relative to the user's home directory. Instead, it seems to be based on whatever Jupyter was started with (so it may be relative to the user's home directory if the jupyter config was defined to be relative to their home dir).

I would just call this `Path to the root directory of the Jupyter server`.

It might also be worth calling it `serverRoot` to maintain consistency with the other references to it.